pom.xml

```diff
@@ -100,16 +100,28 @@
         </exclusion>
       </exclusions>
     </dependency>
-
     <!-- Needed for audit functionality -->
-    <!--
     <dependency>
```

Comment:
I left the dependency in, but the bean is not included in the configuration.xml. I thought that would make it easier to add it in a single step because I don't think it conflicts with anything else. But I can comment it out too.